hail/src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -1550,9 +1550,9 @@ case class MatrixVCFReader(
   override lazy val fullType: TableType = fullMatrixType.canonicalTableType
 
   val fullRVDType = RVDType(PCanonicalStruct(false,
```

Comment:
can this be required true?

Reply:
the top level struct?

Yeah, it could. I was thinking we should go through and make a pass that ensures it's *always* true, because a table row is never missing. Seems like separate work though.